tests/mock_tables/appl_db.json

```diff
@@ -342,5 +342,33 @@
     },
     "BGP_PROFILE_TABLE:FROM_SDN_SLB_ROUTES": {
         "community_id" : "1234:1235"
+    },
+    "P4RT_TABLE:ACL_TABLE_DEFINITION_TABLE:ACL_ACL_PRE_INGRESS_TABLE": {
```

Comment:
>P4RT_TABLE [](http://example.com/codeflow?start=5&length=10)

Could you reformat this json file?

Reply:
Could you please clarify what needs to be reformatted? I run the json file through a json beautifier and I see the same result.

Reply:
I mean the json indentation. I see tabs/spaces mixed.

Reply:
Thanks. I fixed the indentation to use spaces.